charts/keycloak/Chart.yaml

```
@@ -4,7 +4,7 @@ name: keycloak
 version: 0.0.22
```

```suggestion
version: 0.0.23
```

Keycloak v26 :rocket: